spec/helpers/where_with_condition_spec.rb

```diff
@@ -2,43 +2,40 @@
 
 RSpec.describe WhereWithCondition, :type => :helper do
   describe ".where_with_conditions method" do
+
     class TestClass
+      include ActiveRecord::Querying
       include WhereWithCondition
-
-      def where(q)
-      end
     end
 
     let(:test_instance) {TestClass.new}
-    let(:sponsor_active_relation_instance_double) {instance_double Sponsor::ActiveRecord_Relation}
-    let(:orphan_active_relation_instance_double) {instance_double Orphan::ActiveRecord_Relation}
+    let(:active_relation_instance_double) {instance_double Sponsor::ActiveRecord_Relation}
```

Comment:
`WhereWithCondition` should be independent so no need to test it on Orphans also.

`active_relation_instance_double` is just a random Active_Relation instance. (does not matter if it's Sponsor or Orphan...)